Scripts/build.py

```diff
@@ -156,7 +156,7 @@ def xcodebuild(scheme, workspace, config, commands):
 
 
 def check_string_output(command):
-    return subprocess.check_output(command).decode().strip()
+    return subprocess.check_output(command).decode('utf-8').strip()
```

Comment:
I think it wouldn't be a CLI tool that has a problem with the encoding, but `decode` having a default encoding argument of ASCII and throwing if a command outputs non-ASCII text.